database/remotes.mapper.go

```diff
@@ -0,0 +1,328 @@
+package database
+
+// The code below was generated by lxd-generate - DO NOT EDIT!
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"github.com/canonical/microcluster/v2/cluster"
```

Comment:
Manually fixed this one import. Any hints how to `make update-schema` create the file with the right import directly?